tests/non-proof/phase/applyRangedAttack.test.ts

```diff
@@ -73,16 +77,16 @@ describe('PhaseState', () => {
       piecesTree.set(attackingPiece.id.toBigInt(), attackingPiece.hash());
       piecesTree.set(targetPiece1.id.toBigInt(), targetPiece1.hash());
       piecesTree.set(targetPiece2.id.toBigInt(), targetPiece2.hash());
-      gameState = new GameState({
-        piecesRoot: piecesTree.tree.getRoot(),
-        arenaRoot: arenaTree.tree.getRoot(),
-        playerTurn: Field(0),
-        player1PublicKey: player1PrivateKey.toPublicKey(),
-        player2PublicKey: player2PrivateKey.toPublicKey(),
-        arenaLength: UInt32.from(800),
-        arenaWidth: UInt32.from(800),
-        turnsCompleted: UInt32.from(0),
-      });
+      gameState = new GameState(
```

Comment:
Seeing this change to the way the constructor works, I think I understand why, and will revert it in the future for all structs, so their initialization is more legible.